systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaConnectResource.java

```diff
@@ -87,8 +91,8 @@ private static KafkaConnectBuilder defaultKafkaConnect(KafkaConnect kafkaConnect
             .endSpec();
     }
 
-    private static DoneableKafkaConnect deployKafkaConnect(KafkaConnect kafkaConnect) {
-        if (Environment.DEFAULT_TO_DENY_NETWORK_POLICIES) {
+    private static DoneableKafkaConnect deployKafkaConnect(KafkaConnect kafkaConnect, boolean allowNP) {
```

Comment:
Shouldn't be `allowNP` based on env variable for NetowkrPolicies?

Reply:
By default is on `true`, so you'll have to deploy `KafkaClients` pod even if you don't want to send/receive messages or do some requests on Connect's API. From my POV it's better to determine, if we want to deploy KafkaConnect with NP or without, to not create extra Clients pod.

Reply:
make sense. We should try to improve this thing with NP and clients pod, but not in this PR.

Reply:
Yep, that's what I planned, thanks